scripts/run_in_file_program.sh

```diff
@@ -1,5 +1,10 @@
-set -e
+set -eu
 BASE_PATH=$(pwd)
 cd workspace/$CHALLENGE_NAME
-docker run --rm -v $BASE_PATH/challenges/$CHALLENGE_NAME/input.txt:/data/input.txt $CHALLENGE_NAME /data/input.txt  > result.txt
+mkdir data/
+cp $BASE_PATH/challenges/$CHALLENGE_NAME/input.txt data/input.txt
+COMMAND=$(docker inspect prime --format='{{.ContainerConfig.Entrypoint}}')
+CMD=${COMMAND:1:$((${#COMMAND} - 2))} #remove first and last char (long version to work on both Linux and Mac)
+docker run --rm -v "$PWD/data/:/data/" --entrypoint="" $CHALLENGE_NAME  sh -c "$CMD /data/input.txt > /data/result.txt"
```

Comment:
We can also add support for images without shell with a bit of twist:

```
container_id = $(docker create --rm -v "$PWD/data/:/data/" --entrypoint="" $CHALLENGE_NAME injected_busybox sh -c "$CMD /data/input.txt > /data/result.txt")
docker cp /path/to/busybox $container_id:/injected_busybox
docker start $container_id
```

We only need a busy box executable that depends on nothing. You can get it from alpine image.